config/webpack.common.config.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
   entry: {
     assets: path.resolve(__dirname, '../src/index.jsx'),
     accessibilityPolicy: path.resolve(__dirname, '../src/accessibilityIndex.jsx'),
+    i18nMessages: path.resolve(__dirname, '../src/data/i18n/locales/json/currentlySupportedLangs.jsx'),
```

Comment:
So I've come to understand that webpack won't include a file in your project's output unless it appears in an entry point's dependency graph. Since neither `assets ` nor `accessibilityPolicy ` explicitly touch the specific language files, I had to create this dummy entry point to get them into a dependency graph.